wren-ai-service/src/pipelines/generation/followup_sql_generation.py

```diff
@@ -174,6 +175,25 @@ async def post_process(
 ## End of Pipeline
 
 
+class SQLResult(BaseModel):
+    sql: str
+
+
+class GenerationResults(BaseModel):
+    results: list[SQLResult]
+
+
+Generation_MODEL_KWARGS = {
```

Comment:
could this constant be capitalized? please change all xxx_MODEL_KWARGS to all capitalized. thank you

Reply:
done, will be updated.